src/envoy/mixer/utils_test.cc

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2017 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "src/envoy/mixer/utils.h"
+#include "mixer/v1/config/client/client_config.pb.h"
+#include "test/test_common/utility.h"
+
+using Envoy::Http::Utils::ParseJsonMessage;
+
+namespace {
+
+TEST(UtilsTest, ParseNormalMessage) {
+  std::string config_str =
```

Comment:
Use raw string literal (i.e. `R"({...})"` so it is easier to read. JSON keys need `"`s as well.

Reply:
Done